### DISCUSSION TITLE
Function of labelmap

### DISCUSSION THREAD

**Discussion by Matzebhv** [blakeblackshear/frigate#6312] [Ask A Question] [answered]
Hey guys,

if i put together cat and dog via labelmap like this:

model:
  labelmap:
    16: animal
    17: animal

Do i have to delete cat and dog under: 

objects:
      track:
        - person
        - cat
        - dog
        - car

and put animal into config instead?

regards
Matthias

**Comment by NickM-27** [comment, marked as answer]
yes, they need to match

**Reply by Matzebhv** [reply]
Lightning fast answer, thx!
